### ISSUE TITLE
test_rmi on 32 bit:  invalid dims: array size defined by dims is larger than the maximum possible size.

### ISSUE THREAD

**Issue by yarikoptic** [nipy/dipy#1299]
```
======================================================================
ERROR: dipy.tracking.tests.test_utils.test_rmi
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/build/dipy-0.12.0/debian/tmp/usr/lib/python2.7/dist-packages/dipy/tracking/tests/test_utils.py", line 4
72, in test_rmi
    I2 = ravel_multi_index([A, B, C, D], dims=[1000] * 4)
ValueError: invalid dims: array size defined by dims is larger than the maximum possible size.
```
was not sure if 1000 was intentionally large to possibly catch some stats deviations thus shouldn't be generally decreased...

**Comment by arokem** [comment]
See #1300, which potentially solves this issue, by eliminating the whole thing (test and implementation...)

**Comment by arokem** [comment]
Closed through #1300
